src/client/driver/command-executors/execute-navigate-to.js

```diff
@@ -1,19 +1,26 @@
 import hammerhead from '../deps/hammerhead';
-import testCafeCore from '../deps/testcafe-core';
+import { RequestBarrier, pageUnloadBarrier, browser } from '../deps/testcafe-core';
 import DriverStatus from '../status';
 
-var Promise = hammerhead.Promise;
-
-var RequestBarrier    = testCafeCore.RequestBarrier;
-var pageUnloadBarrier = testCafeCore.pageUnloadBarrier;
 
+const { createNativeXHR, utils } = hammerhead;
```

Comment:
const { createNativeXHR, utils, Promise } = hammerhead;

Reply:
There is bug in the hammerhead's eslint rule, it starts to show the `Use Promise from hammerhead` if you use destructuring.

Reply:
I've created issue for this case - https://github.com/LavrovArtem/eslint-plugin-hammerhead/issues/4